File: 3st/Jaeyoung/정수 제곱근 판별.js
function solution(n) {
    var answer = 0;
    for ( var i = 1 ; i <= n ; i++ ) {
        if ( (i*i) == n ) {
            answer = (i+1)*(i+1);
            break;
        }
        else answer = -1;
    }
    return answer;
}

File: 3st/Juram/programmer/정수 제곱근 판별.java
class Solution {
    public long solution(long n) {
        long answer = -1;
        if(Math.sqrt(n)%1==0){
            long s = (long)Math.sqrt(n)+1;
            answer = s*s;
        }
        else answer = -1;
        return answer;
    }
}